docs/en_US/ExperimentConfig.md

```diff
@@ -433,6 +442,14 @@ machineList:
   
     __gpuIndices__ is used to specify designated GPU devices for NNI on this remote machine, if it is set, only the specified GPU devices are used for NNI trial jobs. Single or multiple GPU indices can be specified, multiple GPU indices are seperated by comma(,), such as `1` or  `0,1,3`.
 
+  * __maxTrialNumPerGpu__
+  
+    __maxTrialNumPerGpu__ is used to specify the max concurrency trial number on a GPU device.
+
+  * __useActiveGpu__
+  
+    __useActiveGpu__ is used to specify whether to use a GPU if there is another process. By default, NNI will use the GPU only if there is no another active process in the GPU, if __useActiveGpu__ is set to true, NNI will use the GPU regardless of another processes. This field is not applicable for NNI on Windows.
+
```

Comment:
Nice Job!
I test it on my local machine and it works successfully.
It seems that it also works in remote mode when I catch a sight of the code.

I suggest put one example code here.

```yaml
localConfig:
  maxTrialNumPerGpu:  5
```

Reply:
good suggestion, I've added an configuration in cifar10-pytorch-example